source/core/parameters.cc

```diff
@@ -214,6 +214,11 @@ namespace Parameters
                         Patterns::Double(),
                         "Tracer diffusivity");
 
+      prm.enter_subsection("gravity");
```

Comment:
So this gravity, it is applied on fluid or only on the particles? It's a bit unclear, because we already have a source term section. This will need to be really documented well that this gravity is only the gravity to be used in solid-fluid flows.